packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts

```diff
@@ -0,0 +1,66 @@
+import { render, cleanup } from '@testing-library/svelte';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import { CldOgImage } from '$src/index';
+
+describe('CldImage', () => {
+	afterEach(() => {
+		cleanup();
+	});
+
+    it('should generate the correct URL for Cloudinary Open Graph image in the og:image meta tag', () => {
+        render(CldOgImage, {
+          props: {
+            src: 'og_image',
+
+            config: { cloud: { cloudName: 'testing-og' } },
```

Comment:
could you fix the type errors on these, I believe it's just missing an alt prop which you can like `alt: ''`